client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -119,72 +107,95 @@ module.exports = React.createClass( {
 			}
 		];
 
-		const summaryPageSlug = this.props.site ? this.props.site.slug : '';
-		if ( 'email-followers' === activeFilter ) {
-			summaryPageLink = '/stats/follows/email/' + summaryPageSlug;
-		} else {
-			summaryPageLink = '/stats/follows/wpcom/' + summaryPageSlug;
-		}
-
-		if ( wpcomData && wpcomData.subscribers ) {
-			wpcomFollowers = <StatsList moduleName="wpcomFollowers" data={ wpcomData.subscribers } followList={ this.props.followList } />;
-		}
-
-		if ( emailData && emailData.subscribers ) {
-			emailFollowers = <StatsList moduleName="EmailFollowers" data={ emailData.subscribers } />;
-		}
-
-		if ( wpcomData && wpcomData.total ) {
-			wpcomTotalFollowers = <p>{ this.translate( 'Total WordPress.com Followers' ) }: { this.numberFormat( wpcomData.total ) }</p>;
-		}
-
-		if ( emailData && emailData.total ) {
-			emailTotalFollowers = <p>{ this.translate( 'Total Email Followers' ) }: { this.numberFormat( emailData.total ) }</p>;
-		}
-
-		if ( ( wpcomData && wpcomData.viewAll ) || ( emailData && emailData.viewAll ) ) {
-			viewSummary = (
-				<div key="view-all" className="module-expand">
-					<a href={ summaryPageLink }>{ this.translate( 'View All', { context: 'Stats: Button label to expand a panel' } ) }<span className="right"></span></a>
-				</div>
-				);
-		}
+		const summaryPageSlug = siteSlug || '';
+		const summaryPageLink = 'email-followers' === activeFilter
+			? '/stats/follows/email/' + summaryPageSlug
+			: '/stats/follows/wpcom/' + summaryPageSlug;
 
 		return (
 			<div>
-				<SectionHeader label={ this.translate( 'Followers' ) } href={ summaryPageLink } />
-				<Card className={ classNames.apply( null, classes ) }>
+				<QuerySiteStats statType="statsFollowers" siteId={ siteId } query={ { type: 'wpcom', max: 7 } } />
```

Comment:
On that note, I say we bump this up to max 10 since that would match all other stat modules.  I'm wondering if we did 7 _back in the day_ to conserve horizontal space when stats was in one column 🤔